python/cuml/cuml/accel/estimator_proxy.py

```diff
@@ -357,8 +362,8 @@ def __sklearn_clone__(self):
     ############################################################
 
     @property
-    def _estimator_type(self):
-        return self._cpu._estimator_type
+    def _estimator_type(cls):
+        return cls._cpu._estimator_type
```

Comment:
Would this need to be `@classproperty` ?

Reply:
Ope, `cls` is leftover from trying it as a classproperty (python doesn't actually care about the name `self`, so this still runs as is). Will revert. `classproperty` fixed some bugs and introduced others - the whole `_estimator_type` attribute is going away anyway in sklearn 1.8.